src/jwtUtils.ts

```diff
@@ -124,7 +124,7 @@ function getJwksClient(jwksUri: string) {
   return jwksClient({
     jwksUri,
     cache: true, // Default Value
-    cacheMaxEntries: 5, // Default value
```

Comment:
We increased the number of keys. Increasing `cacheMaxEntries` might prevent performance issues, if we have more than 5 keys. The number 100 was chosen arbitrarily.